docs/docs/gatsby-core-philosophy.md

```diff
@@ -1,8 +1,166 @@
+
+[Read more here](/blog/2017-10-16-making-website-building-fun/).
+
+### Build a better web
+
+Websites can have, or lack, many qualities. They can be fast, secure, maintainable, beautiful, accessible, and cheap (or not). They can be easy to build and iterate on, have great SEO, and fun to work on (or not).
+
+We believe that these qualities should be baked into frameworks, so they are delivered _by default_ rather than implemented on a per-site basis.
+
+When features and qualities are only available if you implement them, that makes them a luxury -- available only to a few websites.
+
+By contrast, by baking in qualities such as performance and security by default, Gatsby makes the right thing the easy thing.
+
+We believe the most high-impact way to make the web better is to make it high-quality by default.
+
+## Gatsby's tooling philosophy
+
+### Bundle the modern Javascript ecosystem for the content web
+
+One key way Gatsby makes the content web high-quality by default is to neatly bundle the modern Javascript ecosystem.
+
+Traditional CMS development [presents many challenges](https://www.gatsbyjs.org/blog/2018-10-11-rise-of-modern-web-development/#traditional-cms-development-presents-challenges) such as walled-garden development, difficult-to-maintain local environments, and a challenging target environment.
+
+Modern web development [bundles advances](https://www.gatsbyjs.org/blog/2018-10-11-rise-of-modern-web-development/#modern-frameworks-offer-stability-and-faster-development) in **performance** (bundle splitting, asset prefetching, offline support, image optimization, or server side rendering), **developer experience** (componentization via React, transpilation via Babel, webpack, hot reloading), **accessibility**, and **security** together.
+
+Gatsby's goal is to bundle these advances in an easy to use package. We're open to any and all advances being made in the modern Javascript world and would love to incorporate them into Gatsby!
+
+For more on this, look at [the features Gatsby bundles together](/features).
+
+### Progressively disclose complexity
+
+One concept UX designers use to make applications easier to learn and less error-prone is called “progressive disclosure".
+
+If you were designing a user interface, you might move advanced or rarely-used features to a secondary screen, such as "advanced settings".
+
+Progressive disclosure simplifies the experience for most people without limiting the abilities of more advanced users.
+
+We progressively disclose complexity by making features such as modifying webpack / Babel config, `gatsby-image` and `gatsby-link` opt-in, simple one-off configuration choices. We avoid all-or-nothing "ejection" scenarios where to add further customization you have to leave the tool behind and manage all complexity (eg, dependencies) yourself.
```

Comment:
[Not using Oxford commas is basically illegal.](https://www.nytimes.com/2017/03/16/us/oxford-comma-lawsuit.html) 😂 

```suggestion
We progressively disclose complexity by making features such as modifying webpack / Babel config, `gatsby-image`, and `gatsby-link` opt-in, simple one-off configuration choices. We avoid all-or-nothing "ejection" scenarios where to add further customization you have to leave the tool behind and manage all complexity (eg, dependencies) yourself.
```

Reply:
@jlengstorf eats shoots and leaves